app/config/tstvhq/imagemaps.json

```diff
@@ -77,368 +73,279 @@
                     "contents": [
                         {
                             "title": "What does it remind you of?",
-                            "path": "https://tstvhq.s3.amazonaws.com/imagemap-content/staircase-map/topic/Mesner\u2014Repressed-Memory.pdf",
-                            "type": "PDF"
```

Comment:
I think you need to restore these `type` lines

Reply:
Okay thanks-- this image map isn't in use to be clear, but I added the missing lines in case it's breaking something else

Reply:
Sorry, you misunderstood: all the lines you deleted that say `"type": "PDF"` need to be restored